src/data/endpoints/managed.yaml

```diff
@@ -238,6 +238,36 @@ endpoints:
             curl -H "Authorization: Bearer $TOKEN" \
                 -X DELETE
                 https://$api_root/$version/managed/services/$service_id
+  /managed/services/$id/enable:
+    group: Services
+    type: Action
+    authenticated: true
+    description: >
+      Enables a Managed service your account may access.
+    methods:
+      POST:
+        description: >
+          Enables this Managed service.
+        examples:
+          curl: |
+            curl -H "Authorization: Bearer $TOKEN" \
+                -X POST
+                https://$api_root/$version/managed/services/$service_id/enable
+  /managed/services/$id/disable:
+    group: Services
+    type: Action
+    authenticated: true
+    description: >
+      Disables a Managed service your account may access.
+    methods:
+      POST:
+        description: >
+          Disables this Managed service.
```

Comment:
maybe add some details about what disabling a service does?  I assume disabled services aren't monitored - you should say so